### PR TITLE
Delete old config that will not be used

### PR DESCRIPTION
**Pull request by smarterclayton**
This is the previous Jenkinsfile work. @cewong @kargakis is any of this in use that you know of?

### CODE REVIEW

File: projects/kubernetes/autoscaler.yaml [hunk 1]
```diff
@@ -1,80 +0,0 @@
-apiVersion: v1
```
Comment (kargakis) [inline projects/kubernetes/autoscaler.yaml]: Don't we need this anymore?

File: projects/kubernetes/cluster-capacity.yaml [hunk 2]
```diff
@@ -1,82 +0,0 @@
-apiVersion: v1
```
Comment (kargakis) [inline projects/kubernetes/cluster-capacity.yaml]: Don't we need this anymore?

File: projects/kubernetes/descheduler.yaml [hunk 3]
```diff
@@ -1,80 +0,0 @@
-apiVersion: v1
```
Comment (kargakis) [inline projects/kubernetes/descheduler.yaml]: Don't we need this anymore?

File: projects/kubernetes/metrics-server.yaml [hunk 4]
```diff
@@ -1,74 +0,0 @@
-apiVersion: v1
```
Comment (kargakis) [inline projects/kubernetes/metrics-server.yaml]: Don't we need this anymore?

**Comment by kargakis** [comment]
Apart from the kubernetes builds, the rest we don't use

cc @stevekuznetsov 

/lgtm
/hold

**Comment by kargakis** [comment]
> Apart from the kubernetes builds

fwiw I don't know whether we use these

**Comment by stevekuznetsov** [comment]
Other than comments from @kargakis LGTM
RIP

File: projects/kubernetes/autoscaler.yaml (hunk 1 above)
Comment (smarterclayton) [reply projects/kubernetes/autoscaler.yaml]: No, we have prow jobs

File: projects/kubernetes/metrics-server.yaml (hunk 4 above)
Comment (smarterclayton) [reply projects/kubernetes/metrics-server.yaml]: No, we have prow jobs

**Comment by kargakis** [comment]
Ok

/hold cancel

**Pull request merged**